### PR TITLE
Make UI responsive

### PR DESCRIPTION
**Pull request by hani-iterable**
## JIRA Ticket(s) if any

* [MOB-7850](https://iterable.atlassian.net/browse/MOB-7850)

## Description

> We have make UI responsible according to figma design

[MOB-7850]: https://iterable.atlassian.net/browse/MOB-7850?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/banner/index.tsx [hunk 1]
```diff
@@ -82,71 +82,100 @@ export const Banner = (props: IBannerProps) => {
...
+              max-height: 2.6em;
+              line-height: 1.3em;
+            }
+            .banner {
+              height: 250px;
+              display: flex;
+              flex-direction: column;
+            }
+          }
+        `}
```
Comment (mprew97) [inline src/components/banner/index.tsx:106]: I'd prefer to keep our styles and our components separate. Please, at a minimum, pull this out into its own constant.

File: src/components/banner/index.tsx [hunk 2]
```diff
@@ -82,71 +82,100 @@ export const Banner = (props: IBannerProps) => {
...
+            flexDirection: 'row'
+          }}
+        >
+          <div style={defaultTextParentStyles}>
+            <text
+              className="titleText"
+              style={{
+                ...defaultTitleStyles,
+                ...titleStyle,
+                display: 'block'
```
Comment (mprew97) [inline src/components/banner/index.tsx:128]: display: block should go into `defaultTitleStyles`

File: src/components/banner/index.tsx [hunk 3]
```diff
@@ -82,71 +82,100 @@ export const Banner = (props: IBannerProps) => {
...
-                    : { ...defaultButtonStyles, ...secondaryBtnStyle }
-                }
-                onClick={onClickSecondaryBtn}
-              >
-                {secondaryBtnLabel ? secondaryBtnLabel : 'Button 2'}
-              </button>
-            ) : null}
+            <text
+              className="titleText"
+              style={{ ...defaultTextStyles, ...textStyle, display: 'block' }}
```
Comment (mprew97) [inline src/components/banner/index.tsx:135]: same here

File: src/components/card/index.tsx [hunk 4]
```diff
@@ -87,59 +87,86 @@ export const Card = (props: ICardProps) => {
...
-              disablePrimaryBtn
-                ? {
-                    ...defaultButtonStyles,
-                    ...primaryDisableBtnStyle
-                  }
-                : { ...defaultButtonStyles, ...primaryBtnStyle }
+    <>
+      <style>
+        {`
+        @media screen and (max-width: 800px) {
```
Comment (mprew97) [inline src/components/card/index.tsx:98]: styles should be in a const

File: src/components/card/index.tsx [hunk 5]
```diff
@@ -87,59 +87,86 @@ export const Card = (props: ICardProps) => {
...
+        style={{ ...defaultCardStyles, ...cardStyle }}
+        onClick={onClickView}
+      >
+        {imgSrc && (
+          <img style={{ ...defaultImageStyles, ...imgStyle }} src={imgSrc} />
+        )}
+        <div style={{ ...defaultTextParentStyles }}>
+          <text
+            className="titleText"
+            style={{ ...defaultTitleStyles, ...titleStyle, display: 'block' }}
```
Comment (mprew97) [inline src/components/card/index.tsx:124]: display: block again

**Review by mprew97 — CHANGES_REQUESTED** [review]

File: src/components/banner/index.tsx (hunk 1 above)
Comment (hani-iterable) [reply src/components/banner/index.tsx]: Done

File: src/components/banner/index.tsx (hunk 2 above)
Comment (hani-iterable) [reply src/components/banner/index.tsx]: Done

File: src/components/banner/index.tsx (hunk 3 above)
Comment (hani-iterable) [reply src/components/banner/index.tsx]: Done

File: src/components/card/index.tsx (hunk 4 above)
Comment (hani-iterable) [reply src/components/card/index.tsx]: Done

File: src/components/card/index.tsx (hunk 5 above)
Comment (hani-iterable) [reply src/components/card/index.tsx]: Done

File: src/components/card/index.tsx [hunk 6]
```diff
@@ -87,59 +87,86 @@ export const Card = (props: ICardProps) => {
...
+          >
+            {title}
+          </text>
+          <text
+            className="titleText"
+            style={{ ...defaultTextStyles, ...textStyle, display: 'block' }}
```
[the remark this answers is not in the document]
Comment (hani-iterable) [reply src/components/card/index.tsx]: Done

File: src/components/notification/index.tsx [hunk 7]
```diff
@@ -68,62 +68,91 @@ export const Notification: React.FC<NotificationProps> = ({
...
-            disablePrimaryBtn
-              ? {
-                  ...primaryButtonDefaultStyle,
-                  ...primaryDisableBtnStyle
-                }
-              : { ...primaryButtonDefaultStyle, ...primaryButtonStyle }
+    <>
+      <style>
+        {`
+        @media screen and (max-width: 800px) {
```
[the remark this answers is not in the document]
Comment (hani-iterable) [reply src/components/notification/index.tsx]: Done

**Pull request merged**